.github/workflows/release.yaml

```diff
@@ -2,8 +2,11 @@ concurrency:
   group: release
   cancel-in-progress: true
 on:
-  pull_request:
-    types: ["closed"]
+  push:
+    branches: [ main ]
+    paths:
+      - lib/**
+      - docs/**
```

Comment:
Only trigger a release if the docs or lib have changed (aka the code has been auto generated again)